src/python/cli_new/lib/cli/http.py

```diff
@@ -19,70 +19,71 @@
 """
 
 import json
-import urllib.request
-import urllib.error
-import urllib.parse
-import time
+from urllib.parse import urlencode
+import urllib3
 
 import cli
 
 from cli.exceptions import CLIException
 
+urllib3.disable_warnings()
 
-def read_endpoint(addr, endpoint, query=None):
+def read_endpoint(addr, endpoint, config, query=None):
     """
     Read the specified endpoint and return the results.
     """
+
+    data = None
```

Comment:
Looking at the code below it doesn't seem that the function can ever return `None` for `data` - maybe no need to set it to `None` here, and just directly return below:
```
return http_response.data.decode('utf-8')
```

Reply:
yes, u are right. :+1: